kubernetes_tests/conftest.py

```diff
@@ -16,11 +16,25 @@
 # under the License.
 from __future__ import annotations
 
+from pathlib import Path
+
 import pytest
 
+DATA_FILES_DIRECTORY = Path(__file__).absolute().parent
```

Comment:
```suggestion
DATA_FILES_DIRECTORY = Path(__file__).resolve().parent
```

I think it's **slightly** better.

Reply:
See that discussion with @uranusjr  :) https://discuss.python.org/t/pathlib-absolute-vs-resolve/2573/19

Reply:
Not really critical, tests will fail on windows way before then come to this import module :rofl:, but let change it for consistency.

To be honest call absolute on `__file__` very soon become redundant, as soon as EOL Python 3.8

Reply:
I've re-readed more carefully, yep `.absolute` have a very interesting implementation an behaviour